ios/ReactNativeCharts/RNChartViewBase.swift

```diff
@@ -105,7 +106,11 @@ open class RNChartViewBase: UIView, ChartViewDelegate {
         }
         
         // TODO extra
-        
+
+        if json["fontFamilyIOS"].string != nil {
```

Comment:
I suppose it's ok to force unwrap the font... going to cause their app to blow up pretty fast during development if someone passes an invalid font family